analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpSyntaxHelper.cs

```diff
@@ -167,23 +167,15 @@ public static bool IsStringEmpty(this ExpressionSyntax expression, SemanticModel
             .Any(symbolPredicate);
     }
 
-    public static SyntaxToken? GetIdentifierOrDefault(this BaseMethodDeclarationSyntax methodDeclaration)
-    {
-        switch (methodDeclaration?.Kind())
+    public static SyntaxToken? GetIdentifierOrDefault(this BaseMethodDeclarationSyntax methodDeclaration) =>
```

Comment:
Not strictly related to unused local functions, but [requested in the same ticket](https://github.com/SonarSource/sonar-dotnet/issues/6699#issuecomment-1413428332).